compiler/AST/type.cpp

```diff
@@ -492,6 +492,7 @@ AggregateType::AggregateType(AggregateTag initTag)
 
   aggregateTag       = initTag;
   initializerStyle   = DEFINES_NONE_USE_DEFAULT;
+  initializerResolved = false;
```

Comment:
It appears that neighboring alignment needs to be updated to match the new variable.